### ISSUE TITLE
Strange bug with new NormalizInterface version

### ISSUE THREAD

**Issue by csoeger** [gap-packages/NormalizInterface#27]
Something is going really wrong, Leo showed it to me and I could reproduce it on my installation (current github versions of gap, NormalizInterface, and Normaliz). It looks like loading NormalizInterface breaks something inside gap.
``` 
gap> G := SmallGroup(192,3);
<pc group of size 192 with 7 generators>
gap> C := CharacterTable(G);
CharacterTable( <pc group of size 192 with 7 generators> )
gap> IsSolvable(C);
true
gap> G := SmallGroup(192,3);
<pc group of size 192 with 7 generators>
gap> C := CharacterTable(G);
CharacterTable( <pc group of size 192 with 7 generators> )
gap> IsSolvable(C);
true
gap> G := SmallGroup(192,3);
<pc group of size 192 with 7 generators>
gap> C := CharacterTable(G);
CharacterTable( <pc group of size 192 with 7 generators> )
gap> IsSolvable(C);
true
gap> LoadPackage("NormalizInterface");
─────────────────────────────────────────────────────────────────────────────────────────────────
Loading  NormalizInterface 0.3 (GAP wrapper for normaliz)
by Sebastian Gutsche (http://wwwb.math.rwth-aachen.de/~gutsche/),
   Max Horn (http://www.quendi.de/math), and
   Christof Söger (http://www.math.uos.de/normaliz).
Homepage: https://github.com/gap-packages/NormalizInterface
─────────────────────────────────────────────────────────────────────────────────────────────────
true
gap> G := SmallGroup(192,3);
<pc group of size 192 with 7 generators>
gap> C := CharacterTable(G);
CharacterTable( <pc group of size 192 with 7 generators> )
gap> IsSolvable(C);
Error, no method found! For debugging hints type ?Recovery from NoMethodFound
Error, no 1st choice method found for `Length' on 1 arguments called from
Length( f ) on line 1086 of file /home/csoeger/gap/lib/pcgsperm.gi called from
CreateIsomorphicPcGroup( pcgs, false, false 
 ) on line 1133 of file /home/csoeger/gap/lib/pcgsperm.gi called from
IsomorphismPcGroup( D.galMorphisms 
 ) on line 1528 of file /home/csoeger/gap/lib/ctblgrp.gi called from
CharacterMorphismGroup( D ) on line 2042 of file /home/csoeger/gap/lib/ctblgrp.gi called from
DixonInit( G, opt ) on line 2160 of file /home/csoeger/gap/lib/ctblgrp.gi called from
...  at line 14 of *stdin*
you can 'quit;' to quit to outer loop, or
you can 'return;' to continue
```
In some other experiments with different orders of execution it even produces a segfault. But never before loading NormalizInterface.

This problem did not exist with the version from Oct, 2nd. So my only guess is the change of TNUM.

**Comment by csoeger** [comment]
I just did a bisect, as I suspected
```
8f5fd0e3c01ef0df01aa32593d62839f37baab81 is the first bad commit
commit 8f5fd0e3c01ef0df01aa32593d62839f37baab81
Author: Max Horn <max@quendi.de>
Date:   Mon Nov 2 15:31:38 2015 +0100

    Use RegisterPackageTNUM to acquire a TNUM for us
    
    This means we now require GAP >= 4.8 (resp. the latest
    GAP development version).
```

**Comment by fingolfin** [comment]
This is a bug in `RegisterPackageTNUM`, I submitted an issue for GAP.

**Comment by fingolfin** [comment]
Fixed the bug in GAP. If you pull the latest git version of GAP, it should be fixed. Hopefull it will also be fixed in the first public release of GAP 4.8.

Thanks a bunch for reporting this, that was a really really embarassing bug.
